### PR TITLE
Add support for Latvian localization

### PR DESCRIPTION
**Pull request by papuass**
Adding support for Latvian localization which I am working on https://translatewiki.net/

Not sure if changes were needed in all of the files (directory layout has changed since last language was added).

### CODE REVIEW

**Comment by papuass** [comment]
OK, as the build is failing, how do I generate Localization/lv.xml ?

File: src/TranslationStats/main.cpp [hunk 1]
```diff
@@ -79,7 +79,8 @@ int main(int argc, char *argv[])
     Localizations::HuggleLocalizations->LocalInit("ko"); // Korean
     Localizations::HuggleLocalizations->LocalInit("ksh");
     Localizations::HuggleLocalizations->LocalInit("lb"); // Lebanon
-    Localizations::HuggleLocalizations->LocalInit("lt");
+    Localizations::HuggleLocalizations->LocalInit("lt"); // Lithuanian
+    Localizations::HuggleLocalizations->LocalInit("lt"); // Latvian
```
Comment (benapetr) [inline src/TranslationStats/main.cpp]: there is "lt" twice

**Comment by benapetr** [comment]
make[2]: *** No rule to make target `/Users/travis/build/huggle/huggle3-qt-lx/src/huggle_l10n/Localization/lv.xml', needed by `huggle_l10n/qrc_text.cpp'.  Stop.

**Comment by benapetr** [comment]
There is no lv.xml file

**Pull request merged**